libcontainer/system/linux.go

```diff
@@ -108,3 +110,13 @@ func GetSubreaper() (int, error) {
 
 	return int(i), nil
 }
+
+// SetLinuxPersonality sets the Linux execution personality. For more information see the personality syscall documentation.
+// checkout getLinuxPersonalityFromStr() from libcontainer/specconv/spec_linux.go for type conversion.
+func SetLinuxPersonality(persona int) error {
+	_, _, errno := syscall.Syscall(syscall.SYS_PERSONALITY, uintptr(persona), 0, 0)
+	if errno != 0 {
+		return fmt.Errorf("syscall: SYS_PERSONALITY failed with error code: %w", errno)
```

Comment:
You can do something like `return &os.SyscallError{Syscall: "set_personality", Err: errno}`, but it's not really important.

Reply:
resolved in latest commit.